src/main/java/com/nix/lesson10/repository/AutoRepository.java

```diff
@@ -54,6 +55,13 @@ public Auto delete(String id) {
         return toDelete;
     }
 
+    @Override
+    public void compare(){
+        autos.sort(new PriceComparator()
+                .thenComparing((o1, o2) -> o1.getModel().compareTo(o2.getModel()))
+                .thenComparing((o1, o2) -> o1.getModel().length() - o2.getModel().length()));
```

Comment:
Сортировка по цене не верная.
Я бы тогда делал все компараторы на анонимных классах
```suggestion
        final Comparator<Vehicle> comparator = Comparator.comparing(Vehicle::getPrice)
                .reversed()
                .thenComparing(Vehicle::getModel)
                .thenComparingInt(o -> o.getModel().length());
        
        autos.sort(comparator);
```

Reply:
Поменял сортировку по цене.
Сделал по цене просто классом для наглядности, что можно итак и через анонимные классы.